### PR TITLE
Making Bearserk Good

### PR DESCRIPTION
**Pull request by TheKillfish**
## What Does This PR Do
Takes Bearserk away from its very conservative original state into a more useful state. Changes are as follows;
- Price reduced to 60 TC
- Pelt gives Heat Resistance when worn (Plasmamen rejoice) and has been made Fireproof so it can survive fires when not worn
- Smokey has been brought in line with the other combos, so you can now regen stamina and get the Bearserker Rage buff
- Harm stamina regen upped to 20
- Combo stamina regen upped to 40
- Stamina regen on Bear Jaws' alt upped to 60
- Bearserk Rage has dedicated icon (improvement could be done, but it's good enough for now)

Previous fix also remains, so because of that, Bearserker has had it's weight swapped with Carp-Fu.

## Why It's Good For The Game
According to Balance Team members that tested it, it was laughably bad and not remotely worth 65 TC. Now it should be better.

## Images of changes
image
Bearserker Rage buff icon. Massive thanks to McRamon for this beautiful sprite

## Testing
Tweaked values, consulted Balance Team, got "lgtm" from some

## Changelog
:cl:
tweak: Tweaked Bearserk so now it's less shit
imageadd: Gave Bearserker Rage an actual sprite
/:cl:

### CODE REVIEW

**Comment by TheKillfish** [comment]
So, PSA; writing `flags = FIRE_PROOF` results in the item becoming NODROP, but not just the Bearserk version; it also applies to the original base version somehow. Going to `resistance_flags = FIRE_PROOF` fixed it.

File: code/modules/martial_arts/combos/bearserk/smokey.dm [hunk 1]
```diff
@@ -13,5 +13,8 @@
 	playsound(get_turf(user), 'sound/weapons/punch1.ogg', 25, vary = TRUE, extrarange = -1)
```
Comment (Burzah) [inline code/modules/martial_arts/combos/bearserk/smokey.dm:13]:
```suggestion
	playsound(get_turf(user), 'sound/weapons/punch1.ogg', 25, TRUE, -1)
```
This should probably be adjusted, while we are here.

**Review by Burzah — CHANGES_REQUESTED** [review]

File: code/modules/martial_arts/bearserk.dm [hunk 2]
```diff
@@ -38,8 +38,10 @@
 
 // The Pelt
 
-/obj/item/clothing/head/bearpelt/bearserk
-	armor = list(MELEE = 30, BULLET = 20, LASER = 20, ENERGY = 20, BOMB = 20, RAD = 0, FIRE = 10, ACID = 10)
+/obj/item/clothing/head/bearpelt/
```
Comment (Henri215) [inline code/modules/martial_arts/bearserk.dm:41]: the path is incomplete?

Comment (TheKillfish) [reply code/modules/martial_arts/bearserk.dm:41]: That would explain some issues. I wonder how long this has been like this... doesn't matter, thanks for noticing!

**Review by Henri215 — APPROVED** [review]
Lewcs Good To Me

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**